tests/zkevm/test_worst_compute.py

```diff
@@ -1720,3 +1720,74 @@ def test_worst_calldataload(
         post={},
         tx=tx,
     )
+
+
+@pytest.mark.parametrize(
+    "opcode",
+    [
+        pytest.param(Op.PUSH0, id="PUSH0"),
```

Comment:
Since it's only one parameter, I think it should print nicely without hardcoding the `id` in each `param`.